topi/python/topi/cuda/softmax.py

```diff
@@ -39,6 +39,7 @@ def schedule_softmax(outs):
     outs = [outs] if isinstance(outs, te.tensor.Tensor) else outs
     s = te.create_schedule([x.op for x in outs])
     softmax = outs[0]
+    tgt = target_.Target.current(allow_none=False)
```

Comment:
We might want to register the warp level strategies only when the target is cuda

Reply:
Hm. I'd like to benefit from the GPU schedules with warps...